core/src/test/scala/org/broadinstitute/dsde/rawls/StatsDTestUtils.scala

```diff
@@ -59,4 +59,15 @@ trait StatsDTestUtils { this: MockitoSugar with Eventually with RawlsTestUtils =
 
   protected def expectedSubmissionStatusMetric(workspaceName: WorkspaceName, submissionStatus: SubmissionStatus, expectedTimes: Int): (String, String) =
     (s"test.workspace.${workspaceName.toString.replace('/', '.')}.submissionStatus.${submissionStatus.toString}", expectedTimes.toString)
+
+  // TODO don't repeat the code from HttpMethodExpansion.makeName() / UriExpansion.makeName()
```

Comment:
I guess I'm advocating hard-coding strings in tests to verify against the actual `uri2String()` logic.